README.md

```diff
@@ -19,15 +31,10 @@ Next install `gym-compete` package as:
 cd gym-compete
 pip install -e .
 ```
-Check install is successful by coming out of the directory and trying `import gym_compete` in python console. Some users might require a `sudo pip install`.
 
-## Trying the environments
-Agent policies are provided for the various environments in folder `agent-zoo`. To see a demo of all the environments do:
+Or use pip to install / add to requirements as
 ```bash
-bash demo_tasks.sh all
+gym_compete @ git+https://github.com/HumanCompatibleAI/multiagent-competition.git@v0.1.0
```

Comment:
Will this install the right branch? I think we might need something like `@mujoco21` (if new branch is called `mujoco21`)

Reply:
Yes, you're right that's what it should be called.